src/run.ts

```diff
@@ -69,6 +81,20 @@ async function runBreaking(): Promise<null|Error> {
             message: 'an against was not provided'
         };
     }
+    const username = core.getInput('buf_input_https_username');
+    if (username === '') {
+        return {
+            message: 'github username not provided'
+        };
+    }
+    process.env[bufInputHTTPSUsername] = username
+    const password = core.getInput('buf_input_https_password');
+    if (password === '') {
+        return {
+            message: 'github token not provided'
```

Comment:
Similar to above. Given that they can be overridden by the user, we shouldn't use the default value naming.